apps/src/music/views/InstrumentGrid/index.tsx

```diff
@@ -1,6 +1,7 @@
 import {SimpleDropdown} from '@code-dot-org/component-library/dropdown';
 import SegmentedButtons from '@code-dot-org/component-library/segmentedButtons';
 import classNames from 'classnames';
+import FocusTrap from 'focus-trap-react';
```

Comment:
Music Lab already uses `react-focus-lock` and (`react-focus-on` by the same author) so it is probably a good idea to stay consistent.

It's my bad for not documenting the exact reason why, but despite [advocating](https://github.com/code-dot-org/code-dot-org/pull/51326#issuecomment-1523216483) for adding `focus-trap-react`, I discovered it didn't handle one of our UI situations in Music Lab properly, whereas `focus-react-lock` did, so we made the switch for this lab.

Reply:
Updated- thanks for the heads up!